localstack/services/sts/sts_starter.py

```diff
@@ -0,0 +1,10 @@
+from localstack.constants import DEFAULT_PORT_STS_BACKEND
```

Comment:
Getting the below error for this import
Unable to register plugins: cannot import name 'DEFAULT_PORT_STS_BACKEND' from 'localstack.constants'